src/cases/services/search-filter.service.ts

```diff
@@ -71,13 +71,16 @@ export class SearchFilterService {
       prefix = parentPrefix + '.';
     }
     for (let attributeName of Object.keys(formGroupValue)) {
-      const value = formGroupValue[attributeName];
+      let value = formGroupValue[attributeName];
       if (isStringOrNumber(value)) {
         const filterType = getFilterType(attributeName, this.metadataFields);
         attributeName = sanitiseMetadataFieldName(filterType, attributeName);
         target[filterType][prefix + attributeName] = value;
       } else if (value) {
-        this.buildFormDetails(attributeName, target, value);
+        if (value[0]) { // is array and has index zero populated
```

Comment:
_Question:_
Why do it this way rather than `if (Array.isArray(value) && value.length > 0)`, which is the standard pattern for checking something is an array of non-zero length?

Reply:
Applied your suggestion

Reply:
Thanks, @adnanakgun. Approved your PR.